src/main/java/seedu/ui/Ui.java

```diff
@@ -105,16 +109,36 @@ public void printListOfIngredients(ArrayList<Ingredient> listOfIngredients, bool
 
     /**
      * Prints a message informing user on list being printed.
+     *
      * @param listOfIngredients The list of ingredients of all items in fridget.
      */
-    public void printListMessage(ArrayList<Ingredient> listOfIngredients) {
-        String printingList = "Here are the list of items in your fridge:";
-        printLine(printingList);
+    public void printListMessage(ArrayList<Ingredient> listOfIngredients, String sortType) {
+        String listMessage = "Here are the list of items in your fridge:\n";
+        listMessage += sortTypeMessage(sortType);
+        printLine(listMessage);
         printListOfIngredients(listOfIngredients, true);
     }
 
+    /**
+     * Determine sortType message.
+     *
+     * @param sortType "e", "r", or "default".
+     * @return sortTypeMessage.
+     */
+    public String sortTypeMessage(String sortType) {
+        switch (sortType) {
+        case "e":
+            return ("< Listing earliest [Expiry Date] first >");
+        case "r":
+            return ("< Listing Most Recently Added items first >");
+        default:
+            return ("< Listing in item [Name] Alphabetical order >");
```

Comment:
I think can change this printing statement to maybe this:
Listing ingredients in alphabetical order.